<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>500</width>
       <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Absorption Coefficient</string>
  </property>
  <widget class="QPushButton" name="graph_button">
   <property name="geometry">
    <rect>
        <x>160</x>
        <y>460</y>
        <width>71</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QComboBox" name="data_name">
   <property name="geometry">
    <rect>
        <x>140</x>
        <y>10</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
        <y>35</y>
        <width>81</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="temperature">
   <property name="geometry">
    <rect>
        <x>140</x>
        <y>36</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
   <property name="value">
    <double>296.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Profile</string>
   </property>
  </widget>
  <widget class="QComboBox" name="graph_type">
   <property name="geometry">
    <rect>
        <x>140</x>
        <y>100</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Voigt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lorentz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gauss</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SD Voigt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Galatry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavenumber Range</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wn_min">
   <property name="geometry">
    <rect>
        <x>110</x>
        <y>180</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wn_max">
   <property name="geometry">
    <rect>
        <x>110</x>
        <y>210</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
        <x>60</x>
        <y>180</y>
     <width>61</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
        <x>60</x>
        <y>210</y>
     <width>61</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
        <x>10</x>
        <y>260</y>
        <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavenumber Step</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wn_step">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
        <y>270</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
        <x>300</x>
        <y>180</y>
        <width>31</width>
        <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;cm&lt;sup&gt;-1&lt;/sup&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
        <x>300</x>
        <y>210</y>
        <width>31</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;cm&lt;sup&gt;-1&lt;/sup&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
        <x>370</x>
        <y>270</y>
        <width>31</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;cm&lt;sup&gt;-1&lt;/sup&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pressure">
   <property name="geometry">
    <rect>
        <x>140</x>
        <y>70</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
        <x>10</x>
        <y>300</y>
        <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavenumber Wing</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wn_wing">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
        <y>310</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
        <x>370</x>
        <y>310</y>
        <width>31</width>
        <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;cm&lt;sup&gt;-1&lt;/sup&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
        <x>330</x>
        <y>40</y>
     <width>21</width>
        <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>K
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
        <x>330</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>atm
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
        <x>10</x>
        <y>340</y>
        <width>141</width>
        <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavenumber Wing Halfwidth</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wn_wing_hw">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
        <y>340</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
        <x>10</x>
        <y>370</y>
        <width>101</width>
        <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity Threshhold</string>
   </property>
  </widget>
     <widget class="QDoubleSpinBox" name="intensity_threshold">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
        <x>180</x>
        <y>370</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
        <x>20</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Broadening Parameter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="broadening_parameter">
   <property name="geometry">
    <rect>
        <x>140</x>
        <y>130</y>
     <width>181</width>
        <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>gamma_air</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gamma_self</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="wn_step_enabled">
   <property name="geometry">
    <rect>
        <x>160</x>
        <y>270</y>
        <width>20</width>
        <height>20</height>
    </rect>
   </property>
      <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
          </sizepolicy>
      </property>
   <property name="text">
    <string/>
   </property>
      <property name="tristate">
          <bool>false</bool>
      </property>
  </widget>
  <widget class="QCheckBox" name="wn_wing_enabled">
   <property name="geometry">
    <rect>
        <x>160</x>
        <y>310</y>
        <width>20</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="wn_wing_hw_enabled">
   <property name="geometry">
    <rect>
        <x>160</x>
        <y>340</y>
        <width>20</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
     <widget class="QCheckBox" name="intensity_threshold_enabled">
   <property name="geometry">
    <rect>
        <x>160</x>
        <y>370</y>
        <width>20</width>
        <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
